lib/user.js

```diff
@@ -11,6 +13,14 @@ var data = require('./data.json');
 module.exports = function(mediator) {
   var store = new UserStore('user');
   return store.init(data).then(function() {
-    return store.listen('', mediator);
+    store.listen('', mediator);
+
+    var dataTopics = new Topics(mediator);
+    dataTopics.prefix('wfm').entity('data')
+    .on('reset', function() {
+      return store.deleteAll().then(function() {
```

Comment:
I don't think the `deleteAll` functionality should be in the user store. It should delete all of the users by publishing delete topics.

Having a deleteAll function in the store means that a small mistake by the user could remove all data from a production system.